docs/modules/player-lifecycle/custom-player-creation.md

```diff
@@ -0,0 +1,127 @@
+")%>
+
+## Manually sending player creation requests
+
+By default, the Player Lifecycle Feature Module automatically sends a player creation request is sent as soon as the client-worker instance connects to SpatialOS.
+
+To modify this behaviour, you must set `PlayerLifecycleConfig.AutoRequestPlayerCreation` to false by one of the following ways:
+
+* Add an extra `false` argument when adding client systems: `PlayerLifecycleHelper.AddClientSystems(Worker.World, autoRequestPlayerCreation: false);`
+* Set the field to false manually during set-up: `PlayerLifecycleConfig.AutoRequestPlayerCreation = false;`
+* Directly modify `PlayerLifecycleConfig` to be false by default
+
+Once `AutoRequestPlayerCreation` is set to false you must manually call the `RequestPlayerCreation` method, which can be found in the `SendCreatePlayerRequestSystem`.
+
+```csharp
+var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+playerCreationSystem.RequestPlayerCreation();
+```
+
+To find out more about the `RequestPlayerCreation` method, take a look at the `SendCreatePlayerRequestSystem` [API reference documentation]({{urlRoot}}/api/player-lifecycle/send-create-player-request-system#methods).
+
+## Adding a player creation callback
+
+If manual player creation is enabled, you can configure a callback to be triggered when a player creation response is received. The callback must be of type `Action<PlayerCreator.CreatePlayer.ReceivedResponse>`.
+
+For example:
+
+```csharp
+void CallPlayerCreation()
+{
+    var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+    playerCreationSystem.RequestPlayerCreation(callback: OnCreatePlayerResponse);
+}
+
+void OnCreatePlayerResponse(PlayerCreator.CreatePlayer.ReceivedResponse response)
+{
+    if (response.StatusCode != StatusCode.Success)
+    {
+        Debug.LogWarning($"Error: {response.Message}");
+    }
+}
+```
+
+## Using arbitrary data in the player creation loop
+
+The default player creation loop sends requests automatically. To use arbitrary data in the loop, you must ensure that `AutoRequestPlayerCreation` is set to false. Some
+
+1. Disable `AutoRequestPlayerCreation`, some ways of doing this are described [here](#manually-sending-player-creation-requests).
+1. Call `RequestPlayerCreation` with serialized data.
+2. Deserialize data back into the original type.
+
+##### Call RequestPlayerCreation with serialized data
+
+Then, call `RequestPlayerCreation`. Ensure that your arbitrary data is serialized into a byte array.
+
+```csharp
+var myArguments = new SampleArgumentsObject
+{
+  PlayerName = "playerName",
+  SpawnPosition = new Coordinates(50, 0, 75))
+};
+var playerCreationSystem = World.GetExistingManager<SendCreatePlayerRequestSystem>();
+var serializedArguments = SerializeArguments(myArguments);
+playerCreationSystem.RequestPlayerCreation(serializedArguments);
+```
+
+Where an example implementation of `SerializeArguments` could be:
```

Comment:
This could be in an expandable with a header of:

"How do I serialize my data?" 

And we can say - there are many ways to serialize data, an example is blah blah blah